yolo_model:

  config_file:
    name: yolov3-tiny-BDT.cfg
  weight_file:
    name: yolov3-tiny-BDT-best.weights
  threshold:
    value: 0.3
  detection_classes:
    names:
      - b
      - uav
      - t
